docs/settings.md

```diff
@@ -3,43 +3,64 @@
 <a name="configuration"></a>
 ## Altering PHP and MySQL configuration
 
-The following configuration files are mounted inside the respective containers and can be used to override the default settings:
+When the following configuration files are added within a project, they can be used to override default settings:
 
-- `.docksal/etc/php5/php.ini` - PHP settings overrides
-- `.docksal/etc/php5/php-cli.ini` - command line PHP settings overrides
+- `.docksal/etc/php/php.ini` - PHP settings overrides
+- `.docksal/etc/php/php-cli.ini` - command line PHP settings overrides
 - `.docksal/etc/mysql/my.cnf` - MySQL settings overrides
 
-Copy [examples/.docksal](../examples/.docksal) into the `/.docksal` folder in your project repo and modify as necessary.
+Copy [examples/.docksal/etc](https://github.com/docksal/docksal/tree/develop/examples/.docksal/etc) into the `/.docksal` folder in your project repo and modify as necessary.
 
 <a name="php-versions"></a>
 ## Using different PHP versions
 
-Switching PHP versions is done via the `blinkreaction/drupal-cli` docker image tags.
+Switching PHP versions is done via the `docksal/drupal-cli` docker image tags. See 
+which images are available on [Docker Hub](https://hub.docker.com/r/docksal/cli/). To
+make these changes you must have project configuration files. See [Project Setup](project-setup.md)
 
 To switch to a different image tag:
 
-- open the `docker-compose.yml` file
-- replace the `cli` service `image` property as necessary (see list of available tags below)
-- run `fin up`
+1 - open the project's `.docksal/docksal.yml` file  
+2 - under the `cli` section, add the `image` property.
+
+By default, it looks like this:
+```
+# CLI node
+  cli:
+    extends:
+      file: ${HOME}/.docksal/stacks/services.yml
+      service: cli
+```
+
+Add the `image` property to override the docker image used:
+```
+# CLI node
+  cli:
+    extends:
+      file: ${HOME}/.docksal/stacks/services.yml
+    image: docksal/cli:1.0-php7
+
+```
+
+See list of available tags below and on [Docker Hub](https://hub.docker.com/r/docksal/cli/tags/).
+
+3 - run `fin up`. This will update the project's configuration.
 
 Available PHP versions:
 
 - `5.6` (`image: docksal/cli:stable`) - default
```

Comment:
cli images which should be used going forward:
- docksal/cli:1.0-php5
- docksal/cli:1.0-php7